ORStools/proc/base_processing_algorithm.py

```diff
@@ -151,14 +188,41 @@ def _get_ors_client_from_provider(provider: str, feedback: QgsProcessingFeedback
         ors_client.overQueryLimit.connect(lambda: feedback.reportError("OverQueryLimit: Retrying..."))
         return ors_client
 
+
+    def parseOptions(self, parameters: dict, context: QgsProcessingContext) -> dict:
+        options = dict()
+
+        if parameters[self.IN_AVOID_FEATS]:
+            options['avoid_features'] = [dict(enumerate(AVOID_FEATURES))[feat] for feat in parameters[self.IN_AVOID_FEATS]]
+
+        if parameters[self.IN_AVOID_BORDERS]:
+            options['avoid_features'] = [dict(enumerate(AVOID_BORDERS))[feat] for feat in parameters[self.IN_AVOID_BORDERS]]
```

Comment:
should we do this as you did with countries and polygons below?
Extract before if and check against and reuse variable for bit more consistency.

Reply:
Yeah, makes sense. I'll unify these a bit :)

Reply:
see above